### ISSUE TITLE
Custom easing has no clear documentation. 

### ISSUE THREAD

**Issue by Alchemist0823** [pixijs/pixi-particles#105]
Greensock CustomEase uses SVG format now.
It doesn't support Greensock CustomEase anymore.

**Comment by andrewstart** [comment]
Yeah, the easing here really needs an overhaul, including a tool that isn't external. It's a long term goal, but in the short term I should be able to make sure it allows for a very basic easing method, so that you can generate it however you want.

**Comment by PotooGryphon** [comment]
Has this feature been updated, since the last response to the thread? I'm also interested in the custom ease feature for the particles

**Comment by andrewstart** [comment]
While I have learned more about bezier curves in the course of my work, I haven't had the time to make any changes - the whole editor needs a rewrite for several reasons. However, the editor should accept a function that takes a number from 0-1 and outputs a number of your choice as an easing method in the config. You wouldn't be able to save it in a JSON file, but you could modify the config at development or runtime to insert a function.

```javascript
// a function like this
function roundEasing(time) {
    return Math.round(time);
}
```

**Comment by PotooGryphon** [comment]
Thank you for the quick response. Looking forward to see any updates with this feature.
